### DISCUSSION TITLE
[Beginner] CSS selector doesn't seem to work?

### DISCUSSION THREAD

**Discussion by Saur-av** [seleniumbase/SeleniumBase#2366] [Q&A] [answered]
I am trying to create a script that logs in to Udemy, But whenever I try to send some keys the module doesn't really find the element. Here's my approach.

```
from seleniumbase import SB

with SB(uc=True) as sb:
    sb.driver.get(r'https://www.udemy.com/join/login-popup/?locale=en_US&response_type=html&next=https%3A%2F%2Fwww.udemy.com%2Flogout%2F')
    sb.type("#form-group--1","test2\n")
    sb.type("#form-group--3","test2\n")
```
Also, I didn't find how to use Xpaths in any of the docs or examples.

**Comment by mdmintz** [comment, marked as answer]
Here's a script to sign in to an existing Udemy account if you provide correct credentials:

```python
from seleniumbase import SB

with SB(uc=True) as sb:
    sb.driver.get("https://www.udemy.com/")
    sb.click('a[data-purpose="header-login"]')
    sb.get_element('[name="email"]').send_keys("user@zz.zzz")
    sb.get_element('[name="password"]').send_keys("PassWord")
    sb.click('button:contains("Log in")')
    breakpoint()
```

**Reply by Saur-av** [reply]
Thanks for your help!
Another quick question, I received a code similar to mine while using `seleniumbase convert {filename.py}` from a recorded script using Katalon. I'm not really sure if it is broken. But I thought it would be best to let you know.

**Reply by mdmintz** [reply]
That's the old way. There's the SeleniumBase Recorder now: https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/recorder_mode.md

Also, the input elements on that page were special because they weren't directly visible without clicking on them first.
